homework_3.py

```diff
@@ -0,0 +1,43 @@
+from flask import Flask
+from sqlalchemy import SQLAlchemy
+from datetime import datetime
+from pydantic import BaseModel
+from datetime import date
+
+app = Flask(__name__)
+
+
+@app.route('/health')
+def health_check():
+    return '', 200
+
+
+db = SQLAlchemy()
+
+
+class AnimalModel(BaseModel):
+    name: str
+    breed: str
+    birthdate: date
+    photo_url: str
+    age: int
```

Comment:
you added age, but I don't see where it will affect response for list.
Also I don't see any changes in html